3_run_rsync/tasks/create-pod-service.yml

```diff
@@ -44,6 +44,31 @@
     src: "{{ mig_dest_ssh_public_key }}"
   register: ssh_public_key
 
+- name: "Get stunnel crt"
+  slurp:
+    src: "{{ stunnel_cert_filepath }}"
+  register: stunnel_crt
+
+- name: "Get stunnel key"
+  slurp:
+    src: "{{ stunnel_key_filepath }}"
+  register: stunnel_key
+
+- name: "Generate rsyncd password"
+  vars:
+    rand_password: "{{ lookup('password', '/dev/null length=15 chars=ascii_letters') }}"
```

Comment:
@alaypatel07 can we directly use `lookup` plugin in `set_fact` task, instead of introducing a new variable `rand_password` ?